README.md

```diff
@@ -10,6 +10,10 @@ The purpose of the open WDI standard is:
 ## Background
 The WDI was initially conceived and developed as part of the National Science Foundation (NSF) Convergence Accelerator Track H project "Mobility Independence through Accelerated Wheelchair Intelligence" (NSF SP0076554) with input from a wide range of stakeholders. 
 
+WDI is intended to fill a gap in current power wheelchair standards. RESNA WC-2 and ISO 7176 do not define or provide guidance on power wheelchair interfaces for third party input devices. And previous attempts to provide a standard for power wheelchair interfaces and accessories (such as the M3S standard) have failed due to lack of industry adoption.
```

Comment:
combine sentences -> "...for third party input devices, and previous attempts..."